cli/support/types/output.ts

```diff
@@ -35,18 +35,28 @@ class Output {
     })
   }
 
+  async outputNotContains(expectedValue: string) {
+    await test.step(`Verify command output contains ${expectedValue}`, async () => {
+      expect(this.stdout, `Stdout does not contain ${expectedValue}!`).not.toContain(expectedValue);
+    })
+  }
+
   async outContainsMany(expectedValues: string[]) {
     for (const val of expectedValues) {
       await test.step(`Verify command output contains ${val}`, async () => {
-        expect.soft(this.stdout,`Stdout does not contain '${val}'!`).toContain(val);
+        expect.soft(this.stdout, `Stdout does not contain '${val}'!`).toContain(val);
       })
     }
     expect(
-        test.info().errors,
-        `'Contains all elements' failed with ${test.info().errors.length} error(s):\n${this.getErrors()}`
+      test.info().errors,
+      `'Contains all elements' failed with ${test.info().errors.length} error(s):\n${this.getErrors()}`
     ).toHaveLength(0);
   }
 
+  async trimWhitespace() {
+    this.stdout = this.stdout.replace(/ +(?= )/g, '');
```

Comment:
Do we really need to "modify" the Output whole object instance? Probably getter (get trimmed stdout) should be a proper solution?
Also why not do to do this in the test body? Do you anticipate it's going to be widely used?
I found it in single usage in "help" validation, maybe we could think of some better name or way to make help validation easy using this approach? We have dedcicated test spec for help: https://github.com/percona/pmm-ui-tests/blob/main/cli/tests/help.spec.ts

Reply:
Fixed

Reply:
great